### PR TITLE
[JAX FE] Add arithmetic operations for JAX frontend.

### PR DESCRIPTION
**Pull request by AsakusaRinne**
### Details:
Add `sub`, `mul` and `div`, and corresponding unit tests.

### Tickets:
No ticket

### CODE REVIEW

**Comment by AsakusaRinne** [comment]
@rkazants This PR added the required arithmetic operations for the resnet, as we discussed yesterday. Could you please help to review it?

**Comment by rkazants** [comment]
Hi @AsakusaRinne, could you please split tests for each arightmetic function to separate function?
based on practice, it is better to manage sporadic issues and manage it.

**Comment by AsakusaRinne** [comment]
> Hi @AsakusaRinne, could you please split tests for each arightmetic operation to separate file? based on practice, it is better to manage sporadic issues and manage it.

Sure, I've separated the test into multiple files.

File: tests/layer_tests/jax_tests/test_div.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+import numpy as np
+import pytest
+
+from jax_layer_test_class import JaxLayerTest
+import jax.numpy as jnp
+
+def get_rand_jnp_array(shape, dtype):
+    if dtype in [np.float32, np.float16, np.float64]:
+        return jnp.array(np.random.uniform(-1000, 1000, shape).astype(dtype))
```
Comment (rkazants) [inline tests/layer_tests/jax_tests/test_div.py:12]: should we avoid division by 0? same comment below

**Review by rkazants — APPROVED** [review]

**Comment by rkazants** [comment]
build_jenkins

File: tests/layer_tests/jax_tests/test_div.py (hunk 1 above)
Comment (rkazants) [reply tests/layer_tests/jax_tests/test_div.py:12]: please exclude this case

**Comment by rkazants** [comment]
build_jenkins

**Pull request merged**